e2e/testsuite/grpc_query.go

```diff
@@ -271,8 +271,8 @@ func (s *E2ETestSuite) QueryProposalV1(ctx context.Context, chain ibc.Chain, pro
 
 // GetBlockHeaderByHeight fetches the block header at a given height.
 func (s *E2ETestSuite) GetBlockHeaderByHeight(ctx context.Context, chain ibc.Chain, height uint64) (Header, error) {
-	cmtservice := s.GetChainGRCPClients(chain).ConsensusServiceClient
-	res, err := cmtservice.GetBlockByHeight(ctx, &ccmtservice.GetBlockByHeightRequest{
+	ccmtservice := s.GetChainGRCPClients(chain).ConsensusServiceClient
```

Comment:
are the diffs here just for the rename? any reason why we do them?

Reply:
ah, conflicts with the import for the client I'd guess `"github.com/cosmos/cosmos-sdk/client/grpc/cmtservice"` and linter is muy triste about it, leaving comment here so others don't ask same thing